pkg/agent/variables.go

```diff
@@ -12,8 +12,6 @@ import (
 	"github.com/blang/semver"
 )
 
-var dockerShimFlags = []string{"--cni-bin-dir", "--cni-cache-dir", "--cni-conf-dir", "--docker-endpoint", "--image-pull-progress-deadline", "--network-plugin", "--network-plugin-mtu"}
```

Comment:
hmm, what happened here?

if this is because you can't have a const slice, i'd prefer a func wrapper to return the locally defined func for the "const-ness" personally

Reply:
This is because lint-pr complained that dockerShimFlags was a global variable. It was only used in `validateAndSetLinuxNodeBootstrappingConfiguration ` so I added the variable inside the function instead. Are you saying that you would rather have a getter function that returns the flags?